rocky/rocky/scheduler.py

```diff
@@ -129,6 +129,18 @@ class Task(BaseModel):
     created_at: datetime.datetime | None = None
     modified_at: datetime.datetime | None = None
 
+    def organization_id(self) -> str:
```

Comment:
This could've been a property perhaps

Reply:
Yeah wasn't sure if pydantic would complain

Reply:
It shouldn't complain unless there's already a field with the same name (like we had with `schema`). Since we commonly use (dynamic) properties in our BaseModels and it's a common approach in Pydantic, this might be inconsistent with the rest, making it worth switching to a property